src/vs/base/browser/ui/grid/grid.ts

```diff
@@ -207,7 +207,12 @@ export namespace Sizing {
 }
 
 export interface IGridStyles extends IGridViewStyles { }
-export interface IGridOptions extends IGridViewOptions { }
+export interface IGridOptions extends IGridViewOptions {
+	/**
+	 * Whether a view is maximized.
+	 */
+	maximizedView?: GridLocation;
```

Comment:
There preference in Grid API is not to expose `GridLocation` at all. Why can't this simply be `maximizedView?: T`, making the interface type be `IGridOptions<T extends IView = IView>`?